crates/conversions/src/byte_array.rs

```diff
@@ -1,9 +1,10 @@
-use crate as conversions; // trick for CairoDeserialize macro
+use crate as conversions;
+// trick for CairoDeserialize macro
```

Comment:
nit:
```suggestion
use crate as conversions; // necessary for CairoDeserialize macro
```
Or maybe even more informative description would be helpful